### PR TITLE
Update ProposalListItem

### PR DESCRIPTION
**Pull request by alisaweb3**
proposalList, contracts modal, contract, rewards, progerss status, teleport

### CODE REVIEW

File: src/modules/[chain]/index.vue [hunk 1]
```diff
@@ -89,6 +89,12 @@ const color = computed(() => {
       return 'text-red-600';
   }
 });
+const endpoint = 'https://juno-api.polkachu.com';
+const params = JSON.stringify({
+  proposal_id: '1',
+  validator_address: 'junovaloper1jxv0u20scum4trha72c7ltfgfqef6nscm9pmg2',
+  chain_name: 'juno',
+});
```
Comment (liangping) [inline src/modules/[chain]/index.vue:97]: Are there added for debug?

File: src/stores/useGovStore.ts [hunk 2]
```diff
@@ -34,12 +38,29 @@ export const useGovStore = defineStore('govStore', {
...
-          proposals?.proposals?.forEach(async (x1) => {
-            await this.fetchTally(x1.proposal_id).then((res) => {
-              x1.final_tally_result = res?.tally;
+          proposals?.proposals?.forEach(async (item) => {
+            await this.fetchTally(item.proposal_id).then((res) => {
+              item.final_tally_result = res?.tally;
+              // 
+             
             });
+            if (this.walletstore.connectedWallet?.cosmosAddress) {
```
Comment (liangping) [inline src/stores/useGovStore.ts:47]: use `walletstore.currentAddress`

File: src/stores/useGovStore.ts [hunk 3]
```diff
@@ -34,12 +38,29 @@ export const useGovStore = defineStore('govStore', {
...
+              try {
+                await this.fetchProposalVotesVoter(item.proposal_id,this.walletstore.currentAddress).then((res) => {
```
Comment (liangping) [inline src/stores/useGovStore.ts:49]: `await` will cause delay for initializing the page.

please separately load  and computed the proposals and votes

File: src/stores/useGovStore.ts [hunk 4]
```diff
@@ -62,5 +83,8 @@ export const useGovStore = defineStore('govStore', {
     async fetchProposalVotes(proposalId: string, next_key?: string) {
```
Comment (liangping) [inline src/stores/useGovStore.ts:83]: GovStore has this function already.

File: src/stores/useGovStore.ts [hunk 5]
```diff
@@ -62,5 +83,8 @@ export const useGovStore = defineStore('govStore', {
...
       return this.blockchain.rpc.getGovProposalVotes(proposalId, next_key);
     },
+    async fetchProposalVotesVoter(proposalId: string, voter: string) {
```
Comment (liangping) [inline src/stores/useGovStore.ts:86]: add it to Gov Store

**Review by liangping — CHANGES_REQUESTED** [review]

File: src/stores/useGovStore.ts (hunk 5 above)
Comment (alisaweb3) [reply src/stores/useGovStore.ts:86]: It looks like it's already in the govStore

**Review by liangping — APPROVED** [review]

**Pull request merged**